docs/user/README.md

```diff
@@ -0,0 +1,38 @@
+# User documentation
+
+## Personal settings
+
+The Assistant personal settings can be found under the "Artificial intelligence" section.
+You can disable the assistant top menu entry there. You can also disable the AI-related smart pickers.
+
+## Run a task
+
+To run a task with the Assistant, just open it via the top menu entry (on the top right).
+
+![top right icon](../img/top.right.button.png)
+
+Then choose a task type at the top of the assistant.
+Then fill up the input form and press the submit button at the bottom right.
+
+![top right icon](../img/assistant.png)
+
+You task will run immediately if possible or be scheduled for later execution.
+
+![top right icon](../img/getting.results.png)
+
+![top right icon](../img/scheduled.png)
+
+## Notifications
+
+If it is scheduled, you will receive a notification when it has finished.
+You can display the task results by clicking the "View results" button in the notification.
+
+![top right icon](../img/notifications.png)
+
+## History
+
+On the bottom left corner of the assistant, there is a button display the task history.
+This task history is filtered to only show tasks of the selected task type.
+You can then relaunch previous tasks, delete them or just cancel them if they were scheduled.
+
+![top right icon](../img/history.png)
```

Comment:
the alt of the images needs an update. lgtm otherwise :tada:

Reply:
:see_no_evil: